snuba/datasets/slicing.py

```diff
@@ -10,12 +10,12 @@
 SENTRY_LOGICAL_PARTITIONS = 256
 
 
-def map_org_id_to_logical_partition(org_id: int) -> int:
+def map_org_id_to_logical_partition(partition_key: int) -> int:
     """
-    Maps an org_id to a logical partition. Since SENTRY_LOGICAL_PARTITIONS is
+    Maps a partition key to a logical partition. Since SENTRY_LOGICAL_PARTITIONS is
     fixed, an org id will always be mapped to the same logical partition.
```

Comment:
this doc comment and the method title still reference org_id, IMO we might just want to remove the rename, but I don't feel strongly

Reply:
this is true. i can revert it and we can revisit later if needed